src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -491,7 +491,15 @@ bool ColorControlServer::stopMoveStepCommand(app::CommandHandler * commandObj, c
     EndpointId endpoint = commandPath.mEndpointId;
     Status status       = Status::Success;
 
-    if (shouldExecuteIfOff(endpoint, optionsMask, optionsOverride))
+    // StopMoveStep command has no effect on an active color loop.
+    // Fetch if it is supported and active.
+    uint8_t isColorLoopActive = 0;
+    if (ColorControlServer::Instance().HasFeature(endpoint, ColorControlServer::Feature::kColorLoop))
```

Comment:
There should be a test for this.  Probably a certification test, ideally, so followup is OK.

Reply:
I will add a steps to the test plan for this then add to the certification test in the SDK
I create a ticket to keep track of it
https://github.com/project-chip/connectedhomeip/issues/34423